build/project-page.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html id="main">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+    <title>Kajsa Unge | Portfolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
```

Comment:
You shouldn't use `maximum-scale=1, user-scalable=0` since it removes the possibility for the user to zoom content if needed. #accessibility :)

Reply:
Makes sense. Would you just remove? Nothing else needed?

Reply:
Makes sense, thanks! Would you just remove? Nothing else needed @PalleZingmark ?

Reply:
Yes @kajsaunge, you can safely remove both `maximum-scale` and `user-scalable`, so you'll end up with this: 
```html
<meta name="viewport" content="width=device-width,initial-scale=1.0"> 
```

Reply:
👍 @PalleZingmark